acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueMiningAcceptanceTest.java

```diff
@@ -72,4 +72,23 @@ public void shouldStallMiningWhenInsufficientValidators() throws IOException {
     minerNode1.verify(net.awaitPeerCount(0));
     minerNode1.verify(clique.noNewBlockCreated(minerNode1));
   }
+
+  @Test
+  public void shouldStillMineWhenSufficientValidators() throws IOException {
+    final PantheonNode minerNode1 = pantheon.createCliqueNode("miner1");
+    final PantheonNode minerNode2 = pantheon.createCliqueNode("miner2");
+    final PantheonNode minerNode3 = pantheon.createCliqueNode("miner3");
+    cluster.start(minerNode1, minerNode2, minerNode3);
+
+    cluster.waitUntil(wait.chainHeadHasProgressedByAtLeast(minerNode1, 1, 85));
+
+    cluster.stopNode(minerNode3);
+    cluster.verifyOnActiveNodes(net.awaitPeerCount(1));
+
+    cluster.waitOnActiveNodesUntil(wait.chainHeadHasProgressedByAtLeast(minerNode1, 1));
+    cluster.verifyOnActiveNodes(clique.proposerAtChainHeadEqual(minerNode1));
```

Comment:
You can't make this guarantee - it would be entirely valid for either minerNode1 or minerNode2 to be the most recent proposer.

Reply:
Yeah your right I can't make that guarantee and it isn't trying to, the method is poorly named. What method is actually doing is verifying a block is created with the given proposer. It doesn't need to the current. I'll think of a better name otherwise it is confusing.